client/src/components/grants/ApplicationCard.tsx

```diff
@@ -85,11 +95,19 @@ export default function ApplicationCard({
       borderWidth="1px"
       borderRadius="md"
     >
-      <Box p={2} mb={4}>
+      <Box p={2} mb={1}>
         <span className="text-[16px] text-gitcoin-gray-400">
           {props.round?.programName}
         </span>
       </Box>
+      <div className="flex justify-end w-fit mr-1 align-middle mb-2">
```

Comment:
we may have to remove this and do on another ticket after design has approved the look/feel of the icon location/placement etc..

Looks good from Eng. standpoint 😁